src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -245,8 +247,15 @@
     <PackageReference Include="System.ServiceModel.Http" Version="4.5.3" />
     <PackageReference Include="System.ServiceModel.NetTcp" Version="4.5.3" />
     <PackageReference Include="System.ServiceModel.Security" Version="4.5.3" />
-
+    
+    
+    
+    <!--System.Text.Json replaces internal NewtonSoft for NET 6-->
+    <Compile Remove="$(PathToMsalSources)\json\**\*.*" />
+    <PackageReference Include="System.Text.Json" Version="7.0.1" />
```

Comment:
I'm a little conflicted about adding this reference here as it may cause customers to get conflicts. @pmaytak - should I add version 6 instead?

Reply:
Yea, I think 6.0.7 is better, and it's the lowest major version that we can use.
Also, I think we should have some of the users test this package out first before merging.